src/test/java/org/folio/circulationbff/api/CheckOutApiTest.java

```diff
@@ -46,10 +52,43 @@ void checkOutSuccess() {
       .andExpect(content().json(mockResponse));
   }
 
```

Comment:
Can we cover all 4 cases of this condition?
`settingsService.isEcsTlrFeatureEnabled() && tenantService.isCurrentTenantCentral()`